dvc/fs/http.py

```diff
@@ -81,11 +81,17 @@ def _prepare_credentials(self, **config):
                     f"Auth method {auth_method!r} is not supported."
                 )
 
-        if "ssl_verify" in config:
-            with fsspec_loop():
+        with fsspec_loop():
+            if "ssl_verify" in config:
                 client_kwargs["connector"] = aiohttp.TCPConnector(
-                    ssl=make_context(config["ssl_verify"])
+                    enable_cleanup_closed=True,
+                    ssl=make_context(config["ssl_verify"]),
                 )
+            else:
+                client_kwargs["connector"] = aiohttp.TCPConnector(
+                    enable_cleanup_closed=True
+                )
+            client_kwargs["connector_owner"] = False
```

Comment:
Note that the connector is closed by `fsspec.implementations.HTTPFilesystem.close_session`